.travis.yml

```diff
@@ -39,6 +39,8 @@ install:
   - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION typhon --file dev-requirements.txt
   # Launch Conda environment
   - source activate test-environment
+  # temporarily install ophyd master
+  - pip install -U git+https://github.com/NSLS-II/ophyd.git
```

Comment:
Might not be a bad idea just to add `lightsource-dev` to the build matrix permanently 🤷‍♂️